deploy.sh

```diff
@@ -170,6 +170,9 @@ localhome=`pwd`
 bucketstackname="RG-PortalStack-Bucket-$runid"
 start_time=$SECONDS
 
+echo "Update Parameter Store"
+updatessmpaths.sh
```

Comment:
Since the updatessmpaths.sh script is not in the PATH, we may need to call this as ./updatessmpaths.sh. Have you tested this?